cadc-rest/src/main/java/ca/nrc/cadc/rest/SyncInput.java

```diff
@@ -359,7 +367,19 @@ private void processStream(String name, String contentType, InputStream inputStr
         }
 
         InlineContentHandler.Content c = inlineContentHandler.accept(name, contentType, new CloseWrapper(inputStream));
-        content.put(c.name, c.value);
+        Object o = content.get(c.name);
+        if (o == null) {
+            content.put(c.name, c.value);
+        } else if (o instanceof List) {
+            List cur = (List) o;
+            cur.add(c);
```

Comment:
Is `content.put(c.name, cur);` missing here?

Reply:
no, the list is already a value in the map, so just add another value to list is sufficient.